yb-voyager/cmd/import.go

```diff
@@ -165,11 +161,9 @@ func validateTargetPortRange() {
 }
 
 func checkOrSetDefaultTargetSSLMode() {
-	if target.Uri == "" {
-		if target.SSLMode == "" {
-			target.SSLMode = "prefer"
-		} else if target.SSLMode != "disable" && target.SSLMode != "prefer" && target.SSLMode != "require" && target.SSLMode != "verify-ca" && target.SSLMode != "verify-full" {
-			utils.ErrExit("Invalid sslmode %q. Required one of [disable, allow, prefer, require, verify-ca, verify-full]", target.SSLMode)
-		}
+	if target.SSLMode == "" {
+		target.SSLMode = "prefer"
+	} else if target.SSLMode != "disable" && target.SSLMode != "prefer" && target.SSLMode != "require" && target.SSLMode != "verify-ca" && target.SSLMode != "verify-full" {
+		utils.ErrExit("Invalid sslmode %q. Required one of [disable, allow, prefer, require, verify-ca, verify-full]", target.SSLMode)
```

Comment:
Prefer switch case over else-if chain (most of the times)
```
switch target.SSLMode {
case "":
  target.SSLMode = "prefer"
case "disable", "prefer", "require", "verify-ca", "verify-full":
  break
default:
  utils.ErrExit("...")
}
```
In general, prefer "positive" conditions instead of "negations"--the former are easier to decode.

Reply:
yeah, I also prefer the positive condition over negations, this was the previous code.